script/build-icons

```diff
@@ -1,11 +1,19 @@
 #!/bin/sh
 
-dir=$(cd $(dirname $0) && cd "../src/common/icons" && pwd)
-src="${dir}/icon.svg"
+dir="$(cd "$(dirname "$0")" && cd "../src/common/icons" && pwd)"
+dark_src="${dir}/icon.svg"
+light_src="${dir}/icon-light.svg"
+
+export_png_icon() {
+  src=$1 dest=$2
+  inkscape -z -o "$dest" -D -w "$s" "$src" # export
+  convert "$dest" -background none -gravity center -extent "${s}x${s}" "$dest" # pad
+  pngcrush -brute -ow "$dest" # reduce
+}
 
 for s in 16 32 48 64 128; do
-  dest="${dir}/icon-${s}.png"
-  inkscape -z -e $dest -D -w $s $src # export
-  convert $dest -background none -gravity center -extent "${s}x${s}" $dest # pad
-  pngcrush -brute -ow $dest # reduce
+  dark_dest="${dir}/icon-${s}.png"
+  light_dest="${dir}/icon-light-${s}.png"
+  export_png_icon "$dark_src" "$dark_dest"
+  export_png_icon "$light_src" "$light_dest"
```

Comment:
As commented above, I think we should explicitly pass the size:

```suggestion
  export_png_icon "$dark_src" "$dark_dest" "$s"
  export_png_icon "$light_src" "$light_dest" "$s"
```

Reply:
Done ✅